content/applications/services/field_service/product_management.rst

```diff
@@ -1,10 +1,30 @@
+==================
+Product management
+==================
+
+When field service workers perform on-site, they commonly use products for the service. Odoo’s
+**Field Service** app allows them to record the products they’re using. Doing so keeps your
+inventory up-to-date in real-time and automatically adds these products to the invoice.
```

Comment:
```suggestion
When field service workers perform tasks on-site, they commonly use products to complete their work. Odoo
Field Service allows them to record these products. Doing so keeps your
inventory up-to-date in real-time and automatically adds the products to the invoice.
```
- to avoid repetitions (use/they're using)
- we usually mention the app either on its own (Field Service) or you can also have Odoo + name of the app, without the possessive form (check out the POS or Accounting doc for reference). You can also have "app" somewhere in there (especially if you want to say "Open the xxx app") but in this case it's not necessary. Bold is also optional for app names (including "Odoo" in this case), but you can keep it if you want.
- Since this is the introduction to the doc page, I think it might be helpful to add a little something about the User default warehous, maybe? Maybe we could move part of the introduction that's under the User dfault warehouse heading here somewhere? I'm not sure, let's discuss this!